src/components/game/wordGuessing/GuessInput.js

```diff
@@ -1,16 +1,75 @@
 import React from 'react';
+import InputField from "../../../views/design/customized-layouts/InputField";
+import Green from "../../../views/design/font-families/Green";
+import styled from "styled-components";
+import Button from "../../../views/design/Button";
+import Red from "../../../views/design/font-families/Red";
 
+const InputGuess = styled(InputField)`
+  &::placeholder {
+  font-family: fantasy;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    letter-spacing: 0.41em;
+    text-transform: uppercase;
+    font-feature-settings: 'cpsp' on, 'ss04' on;
+    color: #00A6EC;
+    mix-blend-mode: darken;
+    text-stroke: 1px #006AAE;
+    -webkit-text-stroke: 1px #006AAE;
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  }
+    text-align:center;
+    font-family: fantasy;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    letter-spacing: 0.41em;
+    text-transform: uppercase;
+    font-feature-settings: 'cpsp' on, 'ss04' on;
+    color: #00A6EC;
+    text-stroke: 1px #006AAE;
+    -webkit-text-stroke: 1px #006AAE;
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+`;
 
 // The input field to submit the guess.
 export class GuessInput extends React.Component {
 
     constructor(props) {
         super(props);
+        this.state = {
+            placeholder:'Enter your guess here...',
```

Comment:
I would move the placeholder into the InputGuess, because it (probably) doesn't change with different states.

Reply:
Yeah, but when I clicked on the input field it doesnt remove the placeholder text untill we type something. So here I used onfocus to remove the placeholder text when someone tries to enter something in the input field. Same applies to clue input.